CHANGES

```diff
@@ -21,6 +21,8 @@ Deprecated
 
 * :confval:`source_parsers` is deprecated. Please use ``add_source_parser()``
   instead.
+* ``app.import_object()`` is deprecated.  Please use
```

Comment:
* ``Application.import_object()`` is deprecated.

Reply:
done